### PR TITLE
add a script to update server manager files

### PR DESCRIPTION
**Pull request by trevj**
Just a quick thing because I got tired of doing this manually. I just used it to generate:
https://github.com/Jigsaw-Code/outline-releases/pull/72

Helps a little with:
https://github.com/Jigsaw-Code/outline-releases/issues/18

### CODE REVIEW

File: manager.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  Outline-Manager.exe.blockmap
+  latest.yml
+  # Linux.
+  Outline-Manager.AppImage
+  latest-linux.yml
+)
+
+function usage() {
+  cat <<-EOM
+Usage: $0 <tag>
```
Comment (alalamav) [inline manager.sh]: Could you add a short explanation of what this script does? A more descriptive filename could also be useful - i.e. release_manager.sh

File: manager.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+pushd manager >/dev/null
+for file in ${FILES[@]}; do
+  echo $file
+  curl -sLO $RELEASE_BASE/$file
+done
+
+git checkout -b manager-$TAG
+git commit -a -m "release server manager $TAG"
+git branch
```
Comment (alalamav) [inline manager.sh]: Bonus points for pushing the newly created branch :)

Comment (trevj) [reply manager.sh]: Done.

File: manager.sh (hunk 1 above)
Comment (trevj) [reply manager.sh]: Both done!

**Comment by trevj** [comment]
@alalamav Thanks for the review, PTAL - I also made the script exit when a download fails.

**Comment by alalamav** [comment]
Thanks for addressing the comments. Are the binaries in outline-server/releases signed? If not, is it worth adding the command to sign them?

**Comment by trevj** [comment]
Yes, they are signed.

**Pull request merged**